### PR TITLE
[13.0][FIX] sale_timesheet_task_exclude self is a recordset

### PR DESCRIPTION
**Pull request by sergiocorato**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@sergiocorato yet you need to fix the precommit

File: sale_timesheet_task_exclude/models/project_task.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def write(self, vals):
         if "exclude_from_sale_order" in vals:
             # If tasks changed their exclude_from_sale_order, update all AALs
             # that have not been invoiced yet
-            for timesheet in self.timesheet_ids.filtered(
+            for timesheet in self.mapped('timesheet_ids').filtered(
```
Comment (alexey-pelykh) [inline sale_timesheet_task_exclude/models/project_task.py]: " instead of ' are used from 13.0+

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

**Comment by sergiocorato** [comment]
> @sergiocorato yet you need to fix the precommit

done, thanks!

**Review by alexey-pelykh — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
@sergiocorato in v15 please just cherry-pick this commit as well

**Comment by alexey-pelykh** [comment]
did that in #534

**Pull request merged**